src/apis/AppState/index.js

```diff
@@ -2,7 +2,7 @@ import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 import findIndex from 'array-find-index';
 import invariant from 'fbjs/lib/invariant';
 
-const EVENT_TYPES = [ 'change' ];
+const EVENT_TYPES = ['change'];
```

Comment:
Wouldn't it be nice to get used to a single format though instead of jumping back and forth when going between companies and projects, like you said? That'd going to happen if we make it an option.